scripts/ci/credscan/CredScanSuppressions.json

```diff
@@ -86,6 +86,12 @@
         "src\\communication\\azext_communication\\tests\\latest\\test_communication_scenario.py"
       ],
       "_justification": "Dummy resources' tokens left during testing Micorosfot.Communication"
+    },
+    {
+      "file": [
+        "src\\virtual-wan\\azext_vwan\\tests\\latest\\recordings\\test_p2s_vpn_client_generate.yaml"
+      ],
+      "_justification": "download URL to non-existing VPN client"
```

Comment:
You should use SAS replacer in test.
FYI
https://github.com/Azure/azure-cli-extensions/blob/549d71be94bc3e31c43fb218a571d5d4bf213ab3/src/application-insights/azext_applicationinsights/tests/latest/recording_processors.py#L19-L46
https://github.com/Azure/azure-cli-extensions/blob/549d71be94bc3e31c43fb218a571d5d4bf213ab3/src/application-insights/azext_applicationinsights/tests/latest/test_applicationinsights_mgmt.py#L13-L17

Reply:
done